src/main/java/com/google/cloud/resourcemanager/ResourceManager.java

```diff
@@ -354,4 +354,15 @@ public static ProjectListOption fields(ProjectField... fields) {
    *     Resource Manager testIamPermissions</a>
    */
   Map<String, Boolean> testOrgPermissions(String resource, List<String> permissions);
+
+  /**
+   * Returns the IAM access control policy for an organization. Returns {@code null} if the resource
```

Comment:
Returning null on permissions issues is strange. I'd expect an exception here.

Reply:
its already thrown an exception in case of failure you can see this method https://github.com/googleapis/java-resourcemanager/pull/122/files#diff-ca644765cbbd814919a4d1ef24fb8eb5R297-R314

here just need delete `Returns {@code null} if the resource does not exist or if you do not have adequate permission to get the policy`